tests/test_transforms.py

```diff
@@ -312,6 +312,15 @@ def test_additional_targets_for_image_only(augmentation_cls, params):
         assert np.array_equal(aug1, aug2)
 
 
+def test_image_invert():
+    # test for np.uint8 dtype
+    image1 = np.random.randint(low=0, high=256, size=(100, 100, 3), dtype=np.uint8)
+    image2 = image1.copy()
+    r1 = F.invert(image1)
+    r2 = F.invert(r1)
+    assert np.array_equal(image2, r2)
```

Comment:
Better to compare uint8 and float32 results

Reply:
Thank you for the feedback. Do you mean test with both uint8 random array and a random float32 array?

Reply:
```python
image2  = A.to_float(image1)
r1 = A.to_float(r1)
```